ios/engine/KMEI/KeymanEngine/KeymanEngine.h

```diff
@@ -15,4 +15,4 @@ FOUNDATION_EXPORT double KeymanEngineVersionNumber;
 FOUNDATION_EXPORT const unsigned char KeymanEngineVersionString[];
 
 // In this header, you should import all the public headers of your framework using statements like #import <KeymanEngine/PublicHeader.h>
-#import <KeymanEngine/Reachability.h>
+//#import <KeymanEngine/Reachability.h>
```

Comment:
Just remove the commented code :)

Reply:
removed

Reply:
on CI, I also had the build fail in that way. Searching the build log came up with watchOS sdk not working (or not found?). Searching for the phrase "This usually indicates that project itself failed to compile" found too many different scenarios for this, but I ended up finding a promising one. It involved this:
  $ unset LLVM_TARGET_TRIPLE_SUFFIX
and updating carthage.
Unfortunately "$ carthage update" fails! :@
But looking at our build script, we don't run that anyway. We update with 
  $ carthage bootstrap --platform iOS
which avoids the failing watchOS build. I think it worked after that.